tests/commands/test_groups.py

```diff
@@ -135,6 +141,18 @@ def test_groups_group_command__create_HTTPError(mock_client, capfd):
     assert "Matching groups (3)" in capture.out
 
 
+def test_groups_group_command__funding_sources(mock_client, capfd):
+    args = Namespace(group_command="funding_sources")
+    mock_client.get_concession_group_linked_funding_sources.return_value = (r for r in GROUP_FUND_RESPONSES)
```

Comment:
This is a nitpick, but I suggest moving this line to come before line 145. This makes it consistent to how other tests set up their mocking before they set up the command arguments (e.g. [line 202-203](https://github.com/cal-itp/littlepay/pull/55/files#diff-4f1378d01e32ec84b6b05dad8ccc0828777b499562e73a3b19d57cd44fb7a452R202-R203))

Reply:
Not at all, I agree, moving it will make it consistent with other tests 👍